tests/devices/test_default_qutrit.py

```diff
@@ -72,6 +72,152 @@ class TestApply:
     """
 
     # TODO: Add tests for non-parametric ops after they're implemented
+    test_data_no_parameters = [
+        (qml.TShift, [1, 0, 0], np.array([0, 1, 0]), None),
+        (
+            qml.TShift,
+            [1 / math.sqrt(2), 1 / math.sqrt(2), 0],
+            np.array([0, 1 / math.sqrt(2), 1 / math.sqrt(2)]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, 1, 0], np.array([0, OMEGA, 0]), None),
+    ]
+
+    test_data_no_parameters_inverses = [
+        (qml.TShift, [0, 1, 0], np.array([1, 0, 0]), None),
+        (
+            qml.TShift,
+            [0, 1 / math.sqrt(2), 1 / math.sqrt(2)],
+            np.array([1 / math.sqrt(2), 1 / math.sqrt(2), 0]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, OMEGA, 0], np.array([0, 1, 0]), None),
+    ]
+
+    @pytest.mark.parametrize("operation, input, expected_output, subspace", test_data_no_parameters)
+    def test_apply_operation_single_wire_no_parameters(
+        self, qutrit_device_1_wire, tol, operation, input, expected_output, subspace
+    ):
+        """Tests that applying an operation yields the expected output state for single wire
+        operations that have no parameters."""
+
+        qutrit_device_1_wire._state = np.array(input, dtype=qutrit_device_1_wire.C_DTYPE)
+        qutrit_device_1_wire.apply(
+            [operation(wires=[0]) if subspace is None else operation(wires=[0], subspace=subspace)]
```

Comment:
This is not a fixture. Some of the future qutrit operations will use subspaces (i.e. will only apply to a subset of the qutrit basis states, and not do anything to the other remaining basis states), so I've written the test this way so that if the `subspace` parameter is not `None` (currently `subsapce` is `None` for all test cases), the `subspace` keyword argument is included in the operation definition. This avoids a lot of repetitive code in the test suite down the line.